pkg/meta/model/reorg.go

```diff
@@ -75,8 +75,9 @@ type DDLReorgMeta struct {
 	// These two variables are used to control the concurrency and batch size of the reorganization process.
 	// They can be adjusted dynamically through `admin alter ddl jobs` command.
 	// Note: Don't get or set these two variables directly, use the functions instead.
```

Comment:
Please make these 3 var private

Reply:
OK, I will try make these vars private in next PR